include/guess/string_util.h

```diff
@@ -1,6 +1,9 @@
 #pragma once
 
 #include <string>
+#include <stdexcept>
+
+#include <utf8proc.h>
```

Comment:
convention in MOTIS is to import non-standard libraries with quotes:

```suggestion
#include "utf8proc.h"
```